clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/config/GoogleInfrastructureProviderConfig.groovy

```diff
@@ -21,22 +21,31 @@ import com.netflix.spectator.api.Registry
 import com.netflix.spinnaker.cats.agent.Agent
 import com.netflix.spinnaker.cats.provider.ProviderSynchronizerTypeWrapper
 import com.netflix.spinnaker.clouddriver.google.GoogleCloudProvider
+import com.netflix.spinnaker.clouddriver.google.GoogleConfiguration
 import com.netflix.spinnaker.clouddriver.google.provider.GoogleInfrastructureProvider
+import com.netflix.spinnaker.clouddriver.google.provider.agent.GoogleLoadBalancerCachingAgent
 import com.netflix.spinnaker.clouddriver.google.provider.agent.GoogleNetworkCachingAgent
 import com.netflix.spinnaker.clouddriver.google.provider.agent.GoogleSecurityGroupCachingAgent
 import com.netflix.spinnaker.clouddriver.google.security.GoogleNamedAccountCredentials
 import com.netflix.spinnaker.clouddriver.security.AccountCredentialsRepository
 import com.netflix.spinnaker.clouddriver.security.ProviderUtils
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.beans.factory.config.ConfigurableBeanFactory
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.DependsOn
+import org.springframework.context.annotation.Import
 import org.springframework.context.annotation.Scope
 
 import java.util.concurrent.ConcurrentHashMap
 
 @Configuration
+@Import(GoogleConfiguration)
 class GoogleInfrastructureProviderConfig {
```

Comment:
Worth noting here that on the aws side, the agents for caching 'infrastructure' resources are configured separately from the application/cluster resources. Here: https://github.com/spinnaker/clouddriver/blob/master/mort/mort-aws/src/main/groovy/com/netflix/spinnaker/mort/aws/provider/config/AwsInfrastructureProviderConfig.groovy vs here: https://github.com/spinnaker/clouddriver/blob/master/oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/config/AwsProviderConfig.groovy

With the merging of oort and mort, we don't necessarily need to do it the same way. But if we do combine the two sets of agents, we probably don't want to name the config class `GoogleInfrastructureProviderConfig`. The choices seem to be:

1) Two provider configs: `GoogleProviderConfig` and `GoogleInfrastructureProviderConfig`
2) One provider config: `GoogleProviderConfig`

I'd vote for simpler, but can live with it either way.